README.md

```diff
@@ -34,27 +34,26 @@ scram build -j 8
 ```bash
 cd PhysicsTools/NanoSUSY/crab
 mkdir -p ${CMSSW_BASE}/src/TopTagger/TopTagger/data
-$CMSSW_BASE/src/TopTagger/TopTagger/scripts/getTaggerCfg.sh -o -n -t DeepResolved_DeepCSV_GR_noDisc_Release_v1.0.0 -d $CMSSW_BASE/src/TopTagger/TopTagger/data
+getTaggerCfg.sh -o -n -t DeepResolved_DeepCSV_GR_noDisc_Release_v1.0.0 -d $CMSSW_BASE/src/TopTagger/TopTagger/data
```

Comment:
We should keep the absolute path to the script, as the current directory is on PhysicsTools/NanoSUSY/crab

Reply:
The script is added to the path as long as you run "cmsenv"

Reply:
OK, sounds good. merging the PR